### ISSUE TITLE
As a SO/CE, I want to be able to have various displays of operating hours content that is updated/maintained in a central place.

### ISSUE THREAD

**Issue by kevincrafts** [CuBoulder/express#1034]
https://github.com/CuBoulder/cu_hours_bundle

## Context

Sites such as UMC, LKibraries, Rec Center, etc need to display operating hours for top level and sub level units. This content ends up in various places on the site in different content entities and different regions.  Because all of these are separate content entities, keeping them up to date can be difficult.

Also, because there are limited ways to build these using tools in Web Express, the solutions are not ideal.

This is for simple displays of operating hours. Scheduling, and handling exceptions for normal operating hours are out of scope for this. 

Example use cases should be typical, consistent operating hours for a business unit, such as a store being open from 8-5 Monday - Friday. This content should not be changing daily or weekly.

## Links to Examples

* http://www.colorado.edu/recreation/
* http://www.colorado.edu/libraries/contact-us/university-libraries-hours
* http://www.colorado.edu/umc/hours

**Comment by kreynen** [comment]
<img width="1217" alt="student_recreation_center_-_google_maps" src="https://cloud.githubusercontent.com/assets/385234/25721375/70c99742-30cd-11e7-8405-3bcd66c119e3.png">

When hours of operation are properly defined, Google Maps picks this up... so I'm not sure we want something that can't be configured to be accurate during breaks/summer/etc

**Comment by cathysnider** [comment]
Initial Review:
https://www-dev.colorado.edu/p1c9ac449cdd/

- Mandatory Hours aren't. I can save an Hours Block wither nor not I've chosen an Hours Set.
- Coordinate with support team to Include a link to 'Hours Tutorial.' (It can be an unpublished, empty page for now. Just set the node number for inclusion here.)
- Change working on Hours tab from 'create hours' to 'Create an Hours Set.' This will make it easier for the support team to talk about 'Hours Blocks' and 'Hours Sets' and how the two are used together. 

![hourschoice](https://user-images.githubusercontent.com/638275/33346599-88231412-d44d-11e7-868c-8e51d3398fcc.jpg)

![hourstab](https://user-images.githubusercontent.com/638275/33346600-884104c2-d44d-11e7-8bd5-92bdb1819138.jpg)

**Comment by cathysnider** [comment]
Am failing this until the small problems are taken care of.

**Comment by cathysnider** [comment]
Acceptance Criteria:
- [ ] Entity Reference works. Example: A Block Row can find Hours Blocks.
